cwf/gateway/integ_tests/gy_enforcement_test.go

```diff
@@ -115,15 +115,15 @@ func provisionRestrictRules(t *testing.T, tr *TestRunner, ruleManager *RuleManag
 	tr.WaitForPoliciesToSync()
 }
 
-// - Set an expectation for a CCR-I to be sent up to OCS, to which it will
-//   respond with a quota grant of 4M.
-//   Generate traffic and assert the CCR-I is received.
-// - Set an expectation for a CCR-U with >80% of data usage to be sent up to
-// 	 OCS, to which it will response with more quota.
-//   Generate traffic and assert the CCR-U is received with final quota grant.
-// - Generate 5M traffic to exceed 100% of the quota and trigger session termination
-// - Assert that UE flows are deleted.
-// - Expect a CCR-T, trigger a UE disconnect, and assert the CCR-T is received.
+//   - Set an expectation for a CCR-I to be sent up to OCS, to which it will
```

Comment:
Why have these extra two spaces been added for lists?

Reply:
This is convention enforced by `gofmt` https://tip.golang.org/doc/comment (scroll down to the heading "Lists"). IntelliJ with the Go plugin reformatted these comments for me automatically pre commit.

Reply:
Moreover, the reason why this was enforced for me is because I use a 1.19 go compiler locally and not 1.18. I suggest keeping the comment changes for this PR and I will downgrade back to 1.18 so that I don't pollute my other PRs with comment formatting going forward. We will eventually have to have a big comment formatting PR once we upgrade past 1.19.